beetsplug/acousticbrainz.py

```diff
@@ -41,64 +45,161 @@ def func(lib, opts, args):
         cmd.func = func
         return [cmd]
 
+    def import_task_files(self, session, task):
+        """Function is called upon beet import.
+        """
+
+        items = task.imported_items()
+        fetch_info(self._log, items)
+
 
 def fetch_info(log, items):
-    """Currently outputs MBID and corresponding request status code
+    """Currently outputs MBID and corresponding request status code.
     """
     for item in items:
         if item.mb_trackid:
             log.info('getting data for: {}', item)
 
             # Fetch the data from the AB API.
-            url = generate_url(item.mb_trackid)
-            log.debug('fetching URL: {}', url)
+            high_url = generate_url(item.mb_trackid, "/high-level")
+            low_url = generate_url(item.mb_trackid, "/low-level")
```

Comment:
You could put both these urls in an array, then loop through the array rather than hard-coding each one separately.